### DISCUSSION TITLE
Audio unit Audiogridder (1.2.0 (BETA 15)  plugin don't work on Ableton Live

### DISCUSSION THREAD

**Discussion by Rikyild** [apohl79/audiogridder#1073] [Help]
Hi all,
Are many versions that the Audio Unit plugin does not work on ableton live, the last one working is in the stable version. 
The problem is tested on several machines with also different OSX and different version of Live.
the VST and VST3 version works without problems. 
the problem is not found on any other DAW, it works correctly in Logic, Studio One, reaper etc.. etc..
I have many sessions that I cannot reopen because I  used the AUDIO UNIT plugin version, but I would like to use the features of these new beta versions.
Are there other people here who have encountered the same problem?
has anyone found a way to bypass this incompatibility?
Many thanks for your help.

<img width="630" alt="Schermata 2022-10-22 alle 12 46 08" src="https://user-images.githubusercontent.com/79607805/197335530-e01e7c18-ea39-4321-a208-27fa434ff244.png">

**Comment by apohl79** [comment]
I can confirm the issue but have not figured out what problem live has, is it does not report any details.

**Reply by Rikyild** [reply]
Ciao Andreas, 
I don't know what problems Ableton can have, but the audio unit plugin is incompatible from the next version to the last stable,
and as I said it is not an isolated problem to my machine, we have tried with more machines, with different versions of Live and with different operating systems (big sur, catalina, high sierra).
The result is always the same, the audio unit version, is not compatible and not initialized, but no crash simply as you can see from the attached picture only reports the problem.
what can i send you to help you in checking if possible?
Many thanks for your help in advance and many thanks for your work.
Enrico

**Reply by apohl79** [reply]
Yeah, I tested that as well and see the problem. Live loads AG, runs the bus configuration tests, and unloads the plugin. In the log Live just says that it failed. No reason whatsoever. So I did a few experiments, but did not find the reason yet. So nothing needed from your end, cheers.

**Reply by Rikyild** [reply]
Many thanks for your effort Andreas.
I hope you can find the reason without wasting too much time.
Have a good weekend
Enrico
